bitnami/metallb/Chart.yaml

```diff
@@ -24,3 +24,8 @@ maintainers:
     email: containers@bitnami.com
 annotations:
   category: Infrastructure
+
+dependencies:
+  - name: common
+    version: 0.x.x
+    repository: https://charts.bitnami.com/bitnami
```

Comment:
To follow the same structure in all of our charts, could you please move this to a new `requirements.yaml`?

Reply:
@FraPazGal i saw #3867 thats why I moved.

Reply:
I agree in moving this now too.